run/rsync_archive/archive-clips.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/bash -eu
 
+# Call wake_up_vehicle Tesla API every 120 seconds
+# as long as rsync is running. This has to be run
+# in the background.  
+function keepVehicleAwake() {
+  if [ -f /root/bin/tesla_api.py ]
+    then
```

Comment:
coding style in other scripts is to indent "then" to the same level as the "if"

Reply:
Makes sense. I have rewritten it to be more generic, and then kill the PID after archive-clips.sh returns. I can submit another pull request, but I can't test the CIFS archive-clips.sh. I have to test a few times first though.

Reply:
It's working so far, but need to do a bit more testing. here is the change:
https://github.com/pgardiner/teslausb/commit/fdae20a006091699c1966e42772d74a0e9c5055c